docs/specification/1.0.md

```diff
@@ -383,35 +383,64 @@ To reduce the implementation burden on EHRs that support CDS services, CDS Hooks
 
-CDS Hooks defines the agreed upon security model between an EHR and the CDS Service. Like SMART on FHIR, the security model of CDS Hooks leverages the same open and well supported standards like OAuth 2 and JSON web tokens. However, as CDS Hooks differs from SMART, the manner in which these standards are used is specific to CDS Hooks.
+CDS Hooks defines a security model that addresses these risks by assuring that the identities of both the CDS Service and the EHR are authenticated to each other; by protecting confidential information and privileged authorizations shared between an EHR and a CDS Service; by recommending means of assuring data freshness; and by incorporating business mechanisms through which trust is established and maintained between an EHR and a CDS Service. 
 
 ### Trusting CDS Services
 
-As the EHR initiates every interaction with the CDS Service, it is responsible for establishing trust with the CDS Services it intends to call. This trust is established via a TLS connection to the CDS Service. Thus, all CDS Service endpoints must be deployed to a TLS protected URL (https). This includes both the Discovery and individual CDS Service endpoints.
+Prior to enabling EHRs to request decision support from any CDS Service, the EHR vendor and/or provider organization is expected to perform due diligence on the CDS Service provider.  Some EHR vendors and provider organizations may maintain a “white list” of CDS Service providers they have deemed trustworthy.  
 
-EHRs should use accepted best practices for verifying the authenticity and trust of these TLS connections. For instance, [rfc5280](https://tools.ietf.org/html/rfc5280) and [rfc6125](https://tools.ietf.org/html/rfc6125). Additionally, it is assumed that EHRs configure the CDS Services they connect to via some offline process according to the business rules and practices of both the EHR and CDS Service organizations.
+Once a CDS Service provider is selected, the EHR vendor/provider negotiates the terms under which service will be provided.  This negotiation includes agreement on patient data elements that will be prefetched and provided to the CDS Service, data elements that will be made available through an access token passed by the EHR, and steps the CDS Service must take to protect patient data and access tokens.  To enable the EHR authorization server to authorize CDS Service access to FHIR resources, the CDS Service is registered as a client to the EHR authorization server.  These business agreements are documented in the service agreement.  Because the CDS Service will receive patient data and will provide clinical decision support, the Service provider also will need to sign a Business Associate Agreement (BAA) that commits it to compliance with the Privacy and Security Rules of the Health Insurance Portability and Accountability Act (HIPAA).   
+
```

Comment:
It seems to me that this is not something the specification needs to be prescriptive about. Perhaps more general language is appropriate here:

The EHR and CDS Service will need to take whatever steps are necessary in their regulatory environments to ensure that the appropriate agreements are in place to cover usage of patient information.

Reply:
@bakerdb - The comment from @brynrhodes is regarding this sentence:

"Because the CDS Service will receive patient data and will provide clinical decision support, the Service provider also will need to sign a Business Associate Agreement (BAA) that commits it to compliance with the Privacy and Security Rules of the Health Insurance Portability and Accountability Act (HIPAA)."

This is a US-centric statement and something we'll want to avoid given that CDS Hooks is a global standard.

Is there a more general, global way of stating this? Or, can we just drop this sentence?

Reply:
@kpshek @brynrhodes  I agree that the sentence re HIPAA is US-specific.  Since a commitment to protect sensitive data is included earlier in this paragraph, I will delete the last sentence.